src/components/pages/Triggers/Triggers.tsx

```diff
@@ -155,37 +141,41 @@ const Triggers: React.FC = () => {
     .flat();
   const events = triggersKeyMap?.events;
 
-  const onSave = (values: any) => {
-    const getSelector = (formValue: any) => {
-      if (typeof formValue === 'string') {
-        if (formValue.includes('/')) {
-          const [namespace, name] = formValue.split('/');
-          return {
-            name,
-            namespace,
-          };
-        }
-        return {
-          name: formValue,
-          namespace: appNamespace,
-        };
-      }
-      if (formValue.length) {
+  const getSelector = (formValue: any) => {
```

Comment:
Based on a quick look at the rest of the code - shouldn't it be:
```suggestion
  const getSelector = (formValue: TestTriggerSelector) => {
```

Reply:
Let's type this a bit later, `TestTriggerSelector` does not fit there. 

<img width="551" alt="image" src="https://user-images.githubusercontent.com/25991946/234846512-29cd177e-cdd1-44f5-bc6f-e44473a0de3d.png">